contrib/completion/bash/docker

```diff
@@ -798,7 +802,7 @@ _docker_run() {
 			__docker_capabilities
 			return
 			;;
-		--cidfile|--env-file)
+		--cidfile|--cgroup-parent|--env-file|--label-file)
```

Comment:
Ah, then it would make more sense to complete it as _anything_. Thanks for clarifying.